packages/model-fragments/tests/integration/app_test.js

```diff
@@ -3,7 +3,7 @@ var app;
 QUnit.module("integration - Application");
 
 test("the model fragments initializer causes no deprecations", function() {
-  expectNoDeprecation();
+  ok(true); // expectNoDeprecation();
```

Comment:
Alright, happy to take a swing at it if you want. We had to build a fake owner in our application's unittest suite to replace the container and registry. We basically followed this: 

```
import FakeOwner from 'ember-getowner-polyfill/fake-owner';
  
export default function buildOwner() {
   let registry = new Ember.Registry();
   let container = new Ember.Container(registry);
   return new FakeOwner({ container: container });
}
```